series/praha-pyvo/events/2020-03-18-testovani.yaml

```diff
@@ -6,7 +6,7 @@ topic: REMOTE PYVO – Testování
 description: |
         **POZOR ZMĚNA:** Vzhledem k postupující nemoci COVID-19 a s tím souvisejícími ochrannými opatřeními se březnové pražské Pyvo uskuteční pouze na dálku. Na Věnečku BUDE ZAVŘENO.
 
-        Březnové remote Pyvo bude mít jako hlavní program testování. Setkáme se online na ZOOMu, https://pantheon.zoom.us/j/233335156.
+        Březnové remote Pyvo bude mít jako hlavní program testování. Setkáme se online na ZOOMu, [https://pantheon.zoom.us/j/233335156](https://pantheon.zoom.us/j/233335156).
         
 venue: remote
 urls: [https://pantheon.zoom.us/j/233335156]
```

Comment:
Udělej to jako seznam:

    urls:
       - https://…

Reply:
Dík!